core/src/test/scala/unit/kafka/server/SaslApiVersionsRequestTest.scala

```diff
@@ -19,13 +19,15 @@ package kafka.server
 import kafka.api.{KafkaSasl, SaslSetup}
 import kafka.security.JaasTestUtils
 import kafka.server.SaslApiVersionsRequestTest.{kafkaClientSaslMechanism, kafkaServerSaslMechanisms}
-import org.apache.kafka.common.test.api.{ClusterTemplate, Type, ClusterTestExtensions, ClusterConfig, ClusterInstance}
+import org.apache.kafka.common.test.api.{ClusterConfig, ClusterTemplate, Type}
 import org.apache.kafka.common.config.SaslConfigs
 import org.apache.kafka.common.config.internals.BrokerSecurityConfigs
 import org.apache.kafka.common.message.SaslHandshakeRequestData
 import org.apache.kafka.common.protocol.{ApiKeys, Errors}
 import org.apache.kafka.common.requests.{ApiVersionsRequest, ApiVersionsResponse, SaslHandshakeRequest, SaslHandshakeResponse}
 import org.apache.kafka.common.security.auth.SecurityProtocol
+import org.apache.kafka.common.test.ClusterInstance
+import org.apache.kafka.common.test.junit.ClusterTestExtensions
```

Comment:
Should we remove `@ExtendWith(value = Array(classOf[ClusterTestExtensions]))` from this class?

Reply:
Wow, good catch 😄  I left this one here intentionally to make sure it still worked if people explicitly add the annotation in the future

Reply:
Well this was a bad choice for a test since this one is disabled. I tried it locally on another test and verified keeping the annotation is harmless